origami/constants.py

```diff
@@ -1,22 +1,40 @@
-ORIGAMI_INJECTION_PATH = "/event"
-ORIGAMI_SERVER_PATH = "localhost:8000"
+ORIGAMI_SERVER_INJECTION_PATH = "/event"
```

Comment:
Shouldn't some of these be in an environment or a configuration file instead of constants?

Reply:
These are default values, the user can change them using the decorators I have added.
It doesn't make sense to have a config file for this(this will increase complexity on users side). The user can always override these values though, so won't be a problem at all.

Reply:
Okay.. :+1: